src/main.ts

```diff
@@ -40,9 +42,24 @@ const init = async (): Promise<void> => {
             forceGraphQLImport: false,
             skipGraphQLImport: true,
         });
+        const schema = makeExecutableSchema({ typeDefs, resolvers });
         const apolloServer = new ApolloServer({
-            typeDefs,
-            resolvers,
+            schema,
+            validationRules: [
+                depthLimit(5),
+                // may need to have resolver level complexity, but this is okay for now.
+                queryComplexity({
+                    maximumComplexity: 1000,
```

Comment:
Looking here: https://typegraphql.ml/docs/0.17.0/complexity.html

I can see this has a `maximumComplexity: 8,  where as we have 1000 what does this number actually mean?

Reply:
We shouldn't make decision on this just yet as we don't have real queries yet. We should tweak it as we approach a realistic looking system. Perhaps the inline comment is not meaningful, I'll make it more explicit. I will reduce the default when switch to config.